buildchain/buildchain/packaging.py

```diff
@@ -211,7 +211,7 @@ def mkdirs() -> None:
     return {
         'title': utils.title_with_target1('GET DEB PKGS'),
         'actions': [mkdirs, dl_packages_callable],
-        'targets': [constants.REPO_DEB_ROOT/'.witness'],
+        'targets': [constants.PKG_DEB_ROOT/'.witness'],
```

Comment:
Commit message should explain why better, like:

```
When building repositories, the packages are copied into the repo hierarchy.
Until now, we were downloading package on the ISO directly (under `_build/root`) but since the repo copy them, we shouldn't download them here (they will be duplicated).
This commit updates the code to download the packages in the "staging" area (under `_build/packages`).
```

Something like that.